builtin/game/chatcommands.lua

```diff
@@ -545,7 +545,14 @@ core.register_chatcommand("spawnentity", {
 			core.log("error", "Unable to spawn entity, player is nil")
 			return false, "Unable to spawn entity, player is nil"
 		end
-		local p = player:getpos()
+		if p == '' then
```

Comment:
These quotes should be double instead of single.  Are you sure there shouldn't be a check here for None as well, though?